.azure-ci.yml

```diff
@@ -68,14 +80,19 @@ jobs:
     #     isOfficialBuild: ${{ and(ne(variables['System.TeamProject'], 'public'), notIn(variables['Build.Reason'], 'PullRequest')) }}
     #     fullMatrix: ${{ notIn(variables['Build.Reason'], 'PullRequest') }}
 
-  # Publish step. Only run in official builds
+  # Publish and validation steps. Only run in official builds
   - ${{ if and(ne(variables['System.TeamProject'], 'public'), notIn(variables['Build.Reason'], 'PullRequest')) }}:
-    - template: /eng/pipelines/publish.yml
+    - template: /eng/pipelines/pre-publish.yml
       parameters:
         dependsOn:
-          - WindowsTest
-          - WindowsNoTest
-          - AllConfigurations
-          - LinuxTest
-          - MacOS
+          - WindowsStage
+          - LinuxStage
+          - MacOSStage
           # - FreeBSD
+
+    - template: eng\common\templates\post-build\post-build.yml
+      parameters:
+        dependsOn:
+          - PrePublish
+        enableSymbolValidation: false # https://github.com/dotnet/arcade/issues/2871
```

Comment:
We need CoreFX issues tracking reenabling of these and need those issues active on 3.0.  I don't want to ship 3.0 without this validation in place.  /cc @markwilkie